lib/DiscoveryStreamFeed.jsm

```diff
@@ -537,8 +554,16 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
   async getComponentFeed(feedUrl, isStartup) {
     const cachedData = await this.cache.get() || {};
     const {feeds} = cachedData;
+    if (!this.timer) {
```

Comment:
This is all just for debugging purposes. It'll be gone once I've finished it up.